src/styles/popup.scss

```diff
@@ -7,17 +7,12 @@
     top: 0px;
     left: 5px;
     .MuiSvgIcon-root{
```

Comment:
why dont you remove this? 
this is done in footer.js:12 thats the place to make modifications for those specific icons

Reply:
so, I removed it and then the language ball broke, then I tried to add css as JSON using `MakeStyles` and I was like why am I using this if I am gonna ended up putting the same CSS, so I left it as it was, also why the two MuiSvgIcon-root are needed I don't know?

Reply:
First you have to understand the difference between the two approaches which I cannot write here as is too much.

Then the answer is: if you leave it like it is and you use other icons in the future, you will have to change it and break all icon styles. 

if you broke the language icon, you have to set specific css to the language icon.

as an example: if you want to change the font size of the footer you don’t change the font size of the whole project. You set a class on the element and change it there otherwise you would be changing the style of the whole project which is not necessary and will bring you problema later on.